warehouse/query-core/src/test/java/datawave/query/QueryPlanTest.java

```diff
@@ -99,7 +102,7 @@ public void planInMetricsAfterInvalidQueryException() throws Exception {
     @Test
     public void planInMetricsAfterMissingIndexException() throws Exception {
         String query = "CITY == 'london' && CITY != 'london'";
-        String expectedPlan = "CITY == 'london' && !(CITY == 'london')";
+        String expectedPlan = "CITY == 'london' && CITY != 'london'";
```

Comment:
why should these have changed?

Reply:
Actually, could we keep this change in addition to the original? That way we test the `not-eq` and `ne` cases

Reply:
I will look at this one.

Reply:
These changed initially after setting the logic to use the federated query planner by default because the ferderated query planner's planned script was not getting updated with the result from `DefaultQueryPlanner.process()` when a RuntimeException was thrown. After correcting this logic, the query planner now reflects the correct query plan. There are now four variants of this test case, two where the logic uses a federated query planner for not-eq and ne, and two where the default query planner is used for not-eq and ne.